web/src/beta/components/fields/SelectField/index.tsx

```diff
@@ -124,6 +124,8 @@ const PickerWrapper = styled(Popover.Content)`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  /* TODO: Fix z-index*/
+  z-index: 700;
```

Comment:
On slack you said zindex isn't needed anymore. Is this deletable?

Reply:
Removed this. It was a in commit that I forgot to push. Good catch!